docusaurus/docs/tutorial/01.md

```diff
@@ -3,213 +3,440 @@ id: 01
+>
+>//without index.js
+>import Todo from "./src/component/todo/todo"
+>```  
+
+3. Write our `todo-list.js` file:
+```javascript
+/* src/component/todo-list/todo-list.js */
 
 import React from "react";
-import Helmet from "@treats/helmet";
-import AsyncComponent from "@treats/component/async-component";
-import style from "./mypage.css";
-
-const pStyle = {
-    fontSize: '12px',
-    textAlign: 'center'
-  };
-  
-const MyPage = () =>(
-    <div className={style.divStyles}>
-        <h1>My Page</h1>
-        <p style={pStyle}>
-            Lorem Ipsum is simply dummy text of
-            the printing and typesetting industry.
-        </p>
-    </div>
-);
 
+const TodoList = ({items}) => (
+    <ul>
+        {items && items.map((item, index) => (
+            <li key={index}>
+                {item}
+            </li>
+        ))}
+    </ul>
+);
 
-export default AsyncComponent(module, MyPage);
+export default TodoList;
 ```
 
-3. Style your page in __`mypage.css`__
+4. (Optional) Style your component in `todo-list.css`.
+
+### The easier way
+Treats can generate components boilerplate using [generators][Generator], so we can jump to step 3. The commands:
+```bash
+// yarn
+~$ yarn generate component
+
+// npm
+~$ npm run generate component
+
+prompt: Component directory:  (./src/component) //directly press enter or change it to another location you preffered
+prompt: Component filename: (my-component) todo-list
+prompt: Component variable name: (MyComponent) TodoList
+prompt: Component description: (My Treats Component) //Fill it with what you want
+
+prompt: Component type (Functional|ClassComponent|ClassPureComponent):  (Functional) //directly press enter for functional component
+
+prompt: Connect your component to redux? (true (t)|false (f)):  (false) //skip it for now, press enter
+prompt: Code split your component (true (t)|false (f)):  (false) //skip it for now, press enter
+
+prompt: Component props (CMD+C to enter): items //enter every props you want. Press CTRL + C or CMD + C when done.
+
+prompt: The above files would be overwritten by the generator are you sure? (true (t) | false (f)):  (false) t
 ```
-.divStyles {
-    margin: 40px;
-    border: 5px dotted pink;
-}
+
+_Voila!_ Find your component in the location you typed and continue to step 3 above.
+
+## Creating Page
+Now that our TodoList component is done, now let's dive into our page.
+
+1. Remove default page `welcome` folder and create new folder named __`todo`__ under `page` folder, which contain __`index.js`__, __`todo.js`__, and __`todo.css`__ under `page` folder. Your folder should look like this:  
+```text
+page
+|-- todo
+    |  
+    |-- index.js
+    |-- todo.js  
+    |-- todo.css  
 ```
+  
+```javascript
+/* index.js */
+import Todo from "./todo";
 
-4. Register a path name for your page in __`./src/_route/path.js`__
+export default Todo;
 ```
-/* ./src/_route/path.js */
 
-....
-export const MYPAGE = "/mypage";
+2. Write __`todo.js`__ for your page
```

Comment:
Hi @martinock great tutorial! I think you can emphasize more on this file __`todo.js`__, to give a grasp on what we want to achieve here, for example: we want to display list of todos, handling state after form submission, etc.